docs/UserGuide.md

```diff
@@ -642,71 +656,106 @@ Important notes:
 
 ###### Example 1
 
-> **Case**: Missing compulsory fields.
+> **Case**: Missing compulsory fields i.e. `INDEX`
 >
 > **Input**: `delete_transaction`
 >
 > **Output**:
 > ```
 > Invalid command format! 
 >
-> delete_transaction: Deletes the transaction identified by the index number used in the displayed transaction list.
+> delete, delete_transaction, del: Deletes the transaction identified by the index number
+> used in the displayed transaction list.
 >
 > Argument: Index (must be a positive integer)
 >
 > Example: delete_transaction 1
 > ```
-> <img src="images/unicash/command-outputs/deleteTransaction/deleteFail1.png" width="1000" />
 
 ###### Example 2
 
-> **Case**: Invalid `<INDEX>` provided
-> (`<INDEX>` given as `10` given when only `5` transactions are presently displayed)
+> **Case**: Invalid `INDEX` provided
+> 
+> **Context**:`INDEX` given as `10` when only `5` transactions are present
+> in the current `Transaction List` configuration.
 >
 > **Input**: `delete_transaction 10`
 >
 > **Output**:
 > ```
 > The transaction index provided is invalid
 > ```
-> <img src="images/unicash/command-outputs/deleteTransaction/deleteFail2.png" width="1000" />
+
+###### Example 3
+
+> **Case**: Invalid `INDEX` provided
+> 
+> **Context**: `INDEX` given as a negative number
+>
+> **Inputs**: `delete_transaction -1`
+>
+> **Output**:
+> ```
+> Invalid command format! 
+>
+> delete, delete_transaction, del: Deletes the transaction identified by the index number
+> used in the displayed transaction list.
+>
+> Argument: Index (must be a positive integer)
+>
+> Example: delete_transaction 1
+> ```
+
+###### Example 4
+
+> **Case**: Invalid `INDEX` provided
+>
+> **Context**: `INDEX` given as a number larger than `Integer.MAX_VALUE`
+>
+> **Inputs**: `delete_transaction 10000000000000000`
+>
+> **Output**:
+> ```
+> Invalid command format! 
+>
+> delete, delete_transaction, del: Deletes the transaction identified by the index number
+> used in the displayed transaction list.
+>
+> Argument: Index (must be a positive integer)
+>
+> Example: delete_transaction 1
+> ```
+
+
 
 #### Get Transaction
 
 Retrieves a `Transaction` from UniCa$h.
 
-Command: `get <INDEX>`
+Command: `get INDEX`
 
-Command Words Accepted: `get`, `g` (case-insensitive)
+Command Words Accepted: `get`, `g` (all case-insensitive)
 
-Command Argument: `<INDEX>` is the displayed transaction index
-of the transaction to be retrieved, as shown in the `Transactions List`.
+Command Argument: `INDEX` is the displayed transaction index
+of the transaction to be deleted, as shown in the currently displayed `Transactions List`.
 
 | Arguments | Optional? | Purpose                                              |
 |-----------|-----------|------------------------------------------------------|
 | `<INDEX>` | No        | Transaction index of the transaction to be retrieved |
```

Comment:
Consistency of omitting `<>` from the INDEX

Reply:
Fixed!